examples/regional-dlp/main.tf

```diff
@@ -29,9 +29,9 @@ module "data_ingestion" {
   sdx_project_number               = var.sdx_project_number
   terraform_service_account        = var.terraform_service_account
   access_context_manager_policy_id = var.access_context_manager_policy_id
-  bucket_name                      = "dlp-flex-data-ingestion"
-  dataset_id                       = "dlp_flex_data_ingestion"
-  cmek_keyring_name                = "dlp_flex_data-ingestion"
+  bucket_name                      = "dlp-flex-data-ing"
+  dataset_id                       = "dlp_flex_data_ing"
+  cmek_keyring_name                = "dlp_flex_data-ing"
```

Comment:
I noticed we are using underscores and hyphens. If the service allows, lets use hyphens?

Reply:
Good suggestion. I used hyphens on `bucket_name` and `cmek_keyring_name`, but on `dataset_id` only underscores are permitted.